test/ssltest.c

```diff
@@ -811,9 +865,7 @@ static void sv_usage(void)
     fprintf(stderr, " -zlib         - use zlib compression\n");
 #ifndef OPENSSL_NO_EC
     fprintf(stderr,
-            " -named_curve arg  - Elliptic curve name to use for ephemeral ECDH keys.\n"
-            "                 Use \"openssl ecparam -list_curves\" for all names\n"
-            "                 (default is sect163r2).\n");
+            " -named_curve arg  - Elliptic curve name to use for ephemeral ECDH keys (obsolete).\n");
```

Comment:
Ditto, this isn't obsolete, but nothing appears to be using it at the moment, so just delete it?